cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -280,6 +278,7 @@ private KafkaAssemblyOperator createCluster(TestContext context) {
             context.assertNotNull(mockClient.secrets().inNamespace(NAMESPACE).withName(KafkaCluster.brokersSecretName(CLUSTER_NAME)).get());
             context.assertNotNull(mockClient.secrets().inNamespace(NAMESPACE).withName(ZookeeperCluster.nodesSecretName(CLUSTER_NAME)).get());
             context.assertNotNull(mockClient.secrets().inNamespace(NAMESPACE).withName(TopicOperator.secretName(CLUSTER_NAME)).get());
+            context.assertNotNull(mockClient.rbac());
```

Comment:
What exactly is this good for?

Reply:
Oh, I guess it is from the test developing phase. Should I remove this? :)

Reply:
It seems to me that it had no actually added value ... so I would remove it.